test/integration/targets/zypper/tasks/zypper.yml

```diff
@@ -1,3 +1,36 @@
+# https://github.com/ansible/ansible/issues/27988
+- name: download apache package to /tmp
+  get_url: 
+    url: http://tube.opensuse.org/repositories/Apache/openSUSE_Tumbleweed/noarch/apache2-doc-2.4.27-124.4.noarch.rpm 
```

Comment:
This URL will break in a few days/weeks/months since old versions are not kept in the repo.
Do you need the .rpm file, or can you just use zypper to talk to the repos directly?

Possible better alternatives:
* use package without version
* use self-build package (see existing tests)

Reply:
I will try with download package from suse repo and install package without version